### PR TITLE
Finished project

### PR DESCRIPTION
**Pull request by MrNopi**

### CODE REVIEW

File: src/main/java/core/basesyntax/Plane.java [hunk 1]
```diff
@@ -7,5 +7,100 @@
...
+
+    public class PlaneBuilder {
+        private String name;
+        private String model;
+        private int maxSpeed;
+        private double maxHeight;
+        private int maxSeats;
+
+        //Getters
+        public double getMaxHeight() {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Plane.java]: remove getters from class PlaneBuilder

File: src/main/java/core/basesyntax/Plane.java [hunk 2]
```diff
@@ -7,5 +7,100 @@
...
+        public String getModel() {
+            return model;
+        }
+
+        public String getName() {
+            return name;
+        }
+
+        // Setters
+        public void setMaxHeight(double maxHeight) {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Plane.java]:
```suggestion
        public PlaneBuilder setMaxHeight(double maxHeight) {
```

File: src/main/java/core/basesyntax/Plane.java [hunk 3]
```diff
@@ -7,5 +7,100 @@
  */
 
 public class Plane {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Plane.java]: try creating object of Plane (for this purpose create main class and method) and write your code in comment under this

File: src/main/java/core/basesyntax/Plane.java (hunk 2 above)
Comment (MrNopi) [reply src/main/java/core/basesyntax/Plane.java]: do you mean that I need to change return type of every setters from void to PlaneBuilder?

File: src/main/java/core/basesyntax/Plane.java (hunk 3 above)
Comment (MrNopi) [reply src/main/java/core/basesyntax/Plane.java]: What do you mean?

File: src/main/java/core/basesyntax/Plane.java [hunk 4]
```diff
@@ -6,6 +6,85 @@
  * которые могут потребоваться для реализации паттерна Builder.</p>
  */
 
-public class Plane {
+public final class Plane {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Plane.java:9]: why did you make this class final? what does `final` keyword in the class stands for?

**Review by Tarabarnik — APPROVED** [review]

**Pull request closed without merging**